docs/configuration.md

```diff
@@ -153,6 +153,7 @@ By default, MMS will use all available GPUs for inference, you use `number_of_gp
 
 Most of those properties are designed for performance tuning. Adjusting those numbers will impact scalability and throughput.
 
+* enable_env_vars_config: Enable configuring MMS through environment variables. When this option is set to "true", all the static configurations of MMS can come through environment variables as well. default: false
```

Comment:
Either change it to `enable_env_vars` to match with the definition in config.properties, or change variable in config.properties to match this name in the doc. Currently they differ by ***_config*** prefix.